### ISSUE TITLE
memory usage

### ISSUE THREAD

**Issue by nezamtrm** [KVishnuVardhanR/M3T#5]
Hi there
Thanks for sharing the code. I just wanted to know how much memory is needed to run the code? and is it possible to run in through colab with its T4 gpu?

**Comment by KVishnuVardhanR** [comment]
Hi, 

Yes, you can run it through colab with its T4 GPU.

**Comment by hczyni** [comment]
> 嗨，感谢您分享代码。我只是想知道运行代码需要多少内存？是否有可能通过其 T4 GPU 通过 colab 运行？
Dear AI developers, can you share the code that executes M3T network? Such as data loading, loss functions, optimizers, etc

**Comment by KVishnuVardhanR** [comment]
Hi,

you can copy the code from the M3T.py file from the repo and call the M3T network using the following command:
`model = M3T()`

Once you called the network, you can use it just like any other network to train the models. But, keep in mind that this network takes in 3D inputs(MRI) of size 128 x 128 x 128.

Thank you
